### ISSUE TITLE
Support for showing authors of file

### ISSUE THREAD

**Issue by ehmkah** [zielu/GitToolBox#27]
It might be useful to find the author who modified the biggest part of a file. For example if you want to known who you should ask for help.

Do you think that such a feature might become a useful part of git-tool-box? Maybe I could also implement it, if you give me a good place where it might fit.

**Comment by zielu** [comment]
Could be useful 👍 . First I would start with git commands to get these statistics: preferably one command but if many then if will also work.
There is also some interesting logic in [git plugin: GitAnnotationProvider.java](https://github.com/JetBrains/intellij-community/blob/master/plugins/git4idea/src/git4idea/annotate/GitAnnotationProvider.java)

**Comment by ehmkah** [comment]
Good idea to ask IntelliJ. Did so lets see what they say: https://youtrack.jetbrains.com/issue/IDEA-165154
